Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.4
+
+RUN apt-get update && \
+  apt-get install -y ruby-full rubygems && \
+  gem install sass
+
+COPY requirements.txt requirements-dev.txt /tock/
+
+RUN pip install -r /tock/requirements.txt -r /tock/requirements-dev.txt
```

Comment:
Note that this isn't being 100% faithful to the [Minimize the number of layers](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#minimize-the-number-of-layers) best practice for Dockerfiles, but I wanted to make it as easy as possible to iterate on the requirements file. Coalescing both `RUN` commands into a single one, while minimizing layers, would also mean that tinkering with requirements would also re-install ruby and sass.